IN-CORE/pyincore
# 310 pandas 200 doesnt work with some of the analysis

To test run below
- tests/pyincore/analyses/ncifunctionality/test_ncifunctionality.py
- tests/pyincore/analyses/combinedwindwavesurgebuildingdamage/test_combinedwindwavesurgebuildingdamage.py
- tests/pyincore/analyses/multiobjectiveretrofitoptimization/test_multiobjectiveretrofitoptimization.py - optional since this runs for hours

Test backwards compatiblity
- [x] pandas 2.0.0
- [x] pandas 1.5.3
- [x] pandas 1.4.1
- [x] pandas 1.3.0
- [x] pandas 1.2.0
- [x] pandas 1.1.0
- [ ] pandas 1.0.0 - doesn't work anymore
AttributeError: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:

- [ ] pandas 0.2.4

Things left to test:

- [x] Combined damage 
- [x] Combined loss
- [x] Multiobject -- runs forever 2+ hours but looks like it's passing all the pandas part
- [x] Transportation is depending on PR #320 

--------------------------
To test:
Run all the major tests in /test folders
